routers/match.py

```
@@ -0,0 +1,16 @@
+from fastapi import APIRouter, Depends, Query, HTTPException
+from fastapi.responses import JSONResponse
+from utils.jwt import get_current_user
+from api.match import get_match_info_by_date, get_match_preview_info
+
+router = APIRouter()
+
+@router.get("/matches")
+def get_matches(date: str = Query(..., description="경기 날짜(YYYY-MM-DD)"), user: dict = Depends(get_current_user)):
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)